### DISCUSSION TITLE
Syntax suggestions for my python library

### DISCUSSION THREAD

**Discussion by ghost** [community/community#65966] [Programming Help]
### Body

from system import mkfile from system import mkdir import system system.mkfile.mkfile("file.txt") system.mkdir.mkdir("f") I am making an extension that is a remake of os and I was wondering how I could make the syntax better mkdir.py: def mkdir(directory): import os if os.path.exists(file): print("The directory already exists!") else: os.mkdir(directory) mkfile.py: def mkfile(file): import os if os.path.exists(file): print("The file already exists!") else: f = open(file, "w") f.close()

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by nirav-gajera** [comment]
* Instead of importing the `mkfile` and `mkdir` modules separately, you can import them as a single module called `system`. This will make your code more concise and easier to read.
* You can also use the `as` keyword to give the `system` module a shorter alias. For example, you could import the module as `sys`.
* Instead of using the `system.mkfile.mkfile()` and `system.mkdir.mkdir()` syntax, you can use the dot notation. This will make your code more readable and easier to maintain. For example, you could write `sys.mkfile("file.txt")` and `sys.mkdir("f")`.
* You can also use the `lambda` function to create a shorter and more concise way to create files and directories. For example, you could write the following code to create a file and a directory:

```python
def create_file(file_name):
    if os.path.exists(file_name):
        print("The file already exists!")
    else:
        f = open(file_name, "w")
        f.close()

def create_directory(directory_name):
    if os.path.exists(directory_name):
        print("The directory already exists!")
    else:
        os.mkdir(directory_name)

create_file("file.txt")
create_directory("f")
```

**Reply by ghost** [reply]
Thanks I'll do that
